pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -661,6 +661,178 @@ func (m *managedFolderLibraryPanelActionsMigrator) Exec(sess *xorm.Session, mg *
 	return nil
 }
 
+const ManagedDashboardAnnotationActionsMigratorID = "managed dashboard permissions annotation actions migration"
+
+func AddManagedDashboardAnnotationActionsMigration(mg *migrator.Migrator) {
+	mg.AddMigration(ManagedDashboardAnnotationActionsMigratorID, &managedDashboardAnnotationActionsMigrator{})
+}
+
+type managedDashboardAnnotationActionsMigrator struct {
+	migrator.MigrationBase
+}
+
+func (m *managedDashboardAnnotationActionsMigrator) SQL(dialect migrator.Dialect) string {
+	return CodeMigrationSQL
+}
+
+// Make this migration run every time until we feel like it's safe to remove it
+func (m *managedDashboardAnnotationActionsMigrator) SkipMigrationLog() bool {
+	return true
```

Comment:
We could also add a config option to run it once.

Reply:
Yup, let's consider if for the future (I looked into it quickly, but could come up with a good fast way to do it).

I've added your suggestion here https://github.com/grafana/identity-access-team/issues/546